connectivity/mbedtls/platform/src/timing_mbed.cpp

```diff
@@ -46,22 +48,25 @@ extern "C" void mbedtls_set_alarm(int seconds)
 
 #if !defined(HAVE_HARDCLOCK)
 #define HAVE_HARDCLOCK
-#include "platform/mbed_rtc_time.h"
 static int hardclock_init = 0;
-static struct timeval tv_init;
 
 extern "C" unsigned long mbedtls_timing_hardclock(void)
 {
-    struct timeval tv_cur;
+#if DEVICE_LPTICKER
+    static mbed::LowPowerTimer timer;
+#elif DEVICE_USTICKER
+    static mbed::Timer timer;
+#else
+#error "MBEDTLS_TIMING_C required either LPTICKER or USTICKER"
```

Comment:
In commit "mbedtls: Use LowPowerTimer/Timer for timing"
```diff
- required
+ requires
```

Reply:
Done

Reply:
Done